src/Workspaces/CSharp/Portable/Extensions/BaseMethodDeclarationSyntaxExtensions.cs

```diff
@@ -39,5 +39,22 @@ public static BaseMethodDeclarationSyntax WithBody(this BaseMethodDeclarationSyn
 
             return node;
         }
+
+        public static BaseMethodDeclarationSyntax WithExpressionBody(this BaseMethodDeclarationSyntax node, ArrowExpressionClauseSyntax expressionBody)
```

Comment:
Note: if this PR gets merged after [https://github.com/dotnet/roslyn/pull/25945](https://github.com/dotnet/roslyn/pull/25945), then the new `With` method here should be removed.

Reply:
Yes, I was just thinking that

Reply:
fixed in a0c5999d887ae519b6cb61e71af684f09e3d47ef